apis/ecr/v1alpha1/doc.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 The Crossplane Authors.
+Copyright 2021 The Crossplane Authors.
```

Comment:
We use the current year in only the files created with that PR.

Reply:
oops